games-fps/prboom-plus/prboom-plus-2.5.1.7.82.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake desktop xdg
+
+MY_PV=$(ver_rs 4 "um+git")
+DESCRIPTION="An enhanced clone of the classic first-person shooter Doom"
+HOMEPAGE="https://github.com/coelckers/prboom-plus/"
+SRC_URI="http://deb.debian.org/debian/pool/main/p/prboom-plus/${PN}_${MY_PV}.orig.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+ GPL-3+ BSD BSD-2 BSD-with-disclosure CC-BY-3.0 CC0-1.0 LGPL-2.1+ MIT public-domain"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="dumb fluidsynth mad +net +opengl +pcre portmidi sdl2-image +sdl2-mixer server vorbis zlib"
```

Comment:
Any particular reason for now enabling `net` by default?

Reply:
PrBoom-Plus is most popular with speedrunners, so I suppose disabling net by default shouldn't be that bad; I'm not sure how popular it is for network play. I'll go ahead and remove the default enable then.